docs/learning/transformer/getting-started.md

```diff
@@ -0,0 +1,87 @@
+# Getting Started with iModel-Transformer
+
+## Overview
+
+The repository supports both JavaScript and TypeScript. To build a strong foundation, check out these resources:
```

Comment:
Is this section on TS/JS necessary?

Reply:
Understood, but if this is going in the transformer package docs, the audience is much larger.